src/Containers/MassTransit.AutofacIntegration/MassTransit.AutofacIntegration.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="Autofac" Version="5.2.0" />
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
```

Comment:
Oh, I'm pretty sure this is going to require me to change Green Pipes and Automatonymous to use the same version. Maybe? Maybe not.

Reply:
there are required more updates in nugets ...